src/xcode/ENA/ENA/Source/Scenes/Events/Store/Model/Event.swift

```diff
@@ -0,0 +1,23 @@
+////
+// 🦠 Corona-Warn-App
+//
+
+enum EventType {
+	case type1
+	case type2
+	case type3
```

Comment:
Can you add some comments or make the cases more precise?

Reply:
At the moment not, maybe in the near future.